<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftwareStatusFrame</class>
 <widget class="QFrame" name="SoftwareStatusFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>200</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
  border: 1px solid gray;
}
QPushButton {
  border-style: none;
  font: 12px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="mainFrameworkLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>主框架部件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataCollectAndBroadcastLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>数据采集与播发部件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataDisposeLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>数据处理部件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataStorageLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>数据存储部件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userManagerLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>用户管理部件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="systemManagerLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>系统监控与管理部件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="mainFrameworkStatus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/icon.qrc">
          <normaloff>:/status_green</normaloff>:/status_green</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataCollectAndBroadcastStatus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/icon.qrc">
          <normaloff>:/status_green</normaloff>:/status_green</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataDisposeStatus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/icon.qrc">
          <normaloff>:/status_red</normaloff>:/status_red</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataStorageStatus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/icon.qrc">
          <normaloff>:/status_green</normaloff>:/status_green</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userManagerStatus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/icon.qrc">
          <normaloff>:/status_green</normaloff>:/status_green</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="systemManagerStatus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/icon.qrc">
          <normaloff>:/status_green</normaloff>:/status_green</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/icon.qrc"/>
 </resources>
 <connections/>
</ui>
